spec/03-plugins/03-http-log/02-schema_spec.lua

```diff
@@ -38,7 +38,8 @@ describe(PLUGIN_NAME .. ": (schema)", function()
     local ok, err = validate({
         http_endpoint = "http://myservice.com/path",
         headers = {
-          ["X-My-Header"] = { "123" }
+          ["X-My-Header"] = "123",
+          ["X-Your-Header"] = "abc",
```

Comment:
I think we should also test an empty header value on these tests.

Reply:
empty headers are not valid